Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -220,11 +221,32 @@ class GitHubEWS(GitHub):
                           ['', 'tv-sim', '', 'jsc-armv7-tests', ''],
                           ['', 'watch', '', '', ''],
                           ['', 'watch-sim', '', '', '']]
-    # FIXME: fetch below user list dynamically and expand it appropriately
-    approved_user_list_for_apple_internal_builds = ['adetaylor', 'aestes', 'aj062', 'annevk', 'aproskuryakov', 'aprotyas', 'beidson', 'briannafan',
-                                       'Constellation', 'danlliu', 'ddkilzer', 'emw-apple', 'eric-carlson', 'etiennesegonzac', 'gsnedders',
-                                       'hortont424', 'jesxilin', 'JonWBedard', 'lilyspiniolas', 'megangardner', 'pxlcoder', 'rr-codes',
-                                       'ryanhaddad', 'Smackteo', 'squelart', 'whsieh', 'zakariaridouh']
+    approved_user_list_for_apple_internal_builds = []
+
+    @classmethod
+    def update_approved_user_list_for_apple_internal_builds(cls):
+        PER_PAGE_LIMIT = 100
+        NUM_PAGE_LIMIT = 10
+        URL_FOR_TRUSTED_GITHUB_LIST = f'https://api.github.com/orgs/apple/teams/webkit/members'
+        members = []
+        for page in range(1, NUM_PAGE_LIMIT + 1):
+            url = f'{URL_FOR_TRUSTED_GITHUB_LIST}?per_page={PER_PAGE_LIMIT}&page={page}'
+            content = cls.fetch_data_from_url_with_authentication_github(url)
+            if not content:
+                _log.error(f'ERROR: Unable to fetch list of trusted users, this might impact Internal EWS.')
```

Comment:
If this happens, is there a step that will propagate up an actionable error message (like contact admin for example)?